Dockerfile

```diff
@@ -18,13 +20,13 @@ RUN yum install -y \
     # Dependencies for building xen \
     checkpolicy gcc python38 python38-devel iasl ncurses-devel libuuid-devel glib2-devel \
     pixman-devel selinux-policy-devel yajl-devel systemd-devel \
-    glibc-devel.i686 glibc-devel \
+    glibc-devel.i686 glibc-devel flex bison wget \
```

Comment:
We need wget for building xen?

Reply:
It is in Xen's own build containers https://xenbits.xen.org/gitweb/?p=xen.git;a=blob;f=automation/build/archlinux/current.dockerfile;h=d46fc9d9ca76ba81e62732ee5eaab0b6bf6f9926;hb=HEAD